app/src/main/res/layout/topic_practice_subtopic.xml

```diff
@@ -33,12 +32,13 @@
       android:id="@+id/subtopic_check_box"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
+      android:paddingBottom="26dp"
       android:button="@drawable/selector_checkbox"
       android:checked="@{isChecked}"
       android:drawableStart="@android:color/transparent"
       android:drawablePadding="20dp"
       android:fontFamily="sans-serif"
-      android:gravity="start"
+      android:gravity="start|top"
       android:minHeight="22dp"
```

Comment:
You can remove `minHeight` from all files fo this view because as we are adding padding bottom the minHeight attribute is of no use now.